pegasus/sites.v3/code.org/public/index.haml

```diff
@@ -6,20 +6,11 @@ video_player: true
 social:
   "og:title": "<%= I18n.t(:og_title) %>"
   "og:description": "<%= I18n.t(:og_description)%>"
-  "og:image": "<%= 'https://' + request.host + '/images/social-media/athletes-thumbnail.jpg' %>"
-  "og:image:width": '1920'
-  "og:image:height": '1080'
-  "og:video:url": "<%= 'http://youtube.com/v/ip051U7Rvds' %>"
-  "og:video:secure_url": "<%= 'https://youtube.com/v/ip051U7Rvds' %>"
-  "og:video:type": "video/mp4"
-  "og:video:width": "640"
-  "og:video:height": "360"
   "twitter:card": player
```

Comment:
I think this can be removed? I thought we include this when we want to show a video player on twitter. Did we test on Twitter?